TMS.Net07.Homework2.DaysOfWeek/TMS.Net07.Homework2.DaysOfWeek/Program.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Threading;
+
+namespace TMS.Net07.Homework2.DaysOfWeek
+{
+    enum DayOfWeek : byte
+    {
+        Sunday,
+        Monday,
+        Tuesday,
+        Wednesday,
+        Thursday,
+        Friday,
+        Saturday
+    }
+
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            while (true)
+            {
+                Console.Write("Input a date before 2100: ");
+                string inputDate = Console.ReadLine(); // Inputted data
```

Comment:
Don't use comments where they are not needed. Clean code should be self-documented. Comments are useful where it's hard to understand what code does